tests/type_traits.cpp

```diff
@@ -57,11 +57,150 @@ GTEST("ozo::is_null<boost::weak_ptr>()", "[for uninitialized pointer returns fal
     EXPECT_TRUE(ozo::is_null(ptr));
 }
 
-
 GTEST("ozo::is_null<not nullable>()", "[for int returns false]") {
     EXPECT_TRUE(!ozo::is_null(int()));
 }
 
+GTEST("ozo::unwrap_nullable") {
+    int wrapped_value = 7;
+
+    SHOULD("unwrap boost::optional") {
+        boost::optional<int> n(wrapped_value);
+        EXPECT_EQ(ozo::unwrap_nullable(n), wrapped_value);
```

Comment:
Explicit value as the expectation is better than variable, especially when it used only once